src/com/exadel/bootcamp/CommandExecutor.java

```diff
@@ -102,66 +67,83 @@ public static void runAdmin() {
                     History.setHistory("checkout stats");
                 } else if (command.equals("users")) {
                     Users.showUsers();
+                    History.setHistory("checkout users list");
+                } else if (command.equals("cash")) {
+                    System.out.println("What name of cash machine?");
+                    CashMachines.getCashMachine(scanner.nextLine()).getAllBanknotesSum();
                 }
             }
          } catch(Exception e){
```

Comment:
Another gentlemen's rule, is to specify an exception more precisely than just "Exception". 

An exception is just too abstract. When you catch `Exception`, this is like you tell "Guys, something bad can happen here, I don't know what, sorry". While the real mistakes have their names, like `IOException`, `NullPointerException`, and so on.

When working on a professional project, please always try to *investigate* your own code and your colleagues' code, find out what exactly can happen, and in what circumstances, and why.